example/eventGenerator/src/EventTelemetry.js

```diff
@@ -22,24 +22,24 @@
 /*global define */
 
 /**
- * Module defining EventTelemetry. 
+ * Module defining EventTelemetry.
  * Created by chacskaylo on 06/18/2015.
- * Modified by shale on 06/23/2015. 
+ * Modified by shale on 06/23/2015.
  */
 define(
-    ['text!../data/transcript.json'],
-    function (transcript) {
+    ['../data/transcript.json'],
+    function (messages) {
         "use strict";
 
-        var firstObservedTime = Date.now(),
-            messages = JSON.parse(transcript);
-        
+        var firstObservedTime = Date.now();
+            // messages = JSON.parse(transcript);
```

Comment:
Let's remove this line now that webpack is magically providing parsed javascript objects.